classes/capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class CapitalizeDecorator < Decorator
```

Comment:
You inherited from the class `Decorator` however, the class `Decorator` (Base Decorator) is missing in your project source directory. Kindly ensure the class Decorator is present and it should have the following features / properties:
- [ ] `Decorator` class should inherits from `Namable` class.
- [ ] Should have a constructor a namable object from params is assigned to an instance variable.
- [ ] The `correct_name` method that returns the result of the `correct_name` method of the `@namable` is implemented.